### ISSUE TITLE
Toggle syntax not working when item is UNDEF or NULL

### ISSUE THREAD

**Issue by jimtng** [boc-tothefuture/openhab-jruby#20]
This syntax:
```ruby
SwitchItem << !SwitchItem
```
was meant to toggle an item ON/OFF. However when the item is in UNDEF or NULL state, it won't work. It gave me this:

```
10:11:20.937 [WARN ] [ab.core.model.script.actions.BusEvent] - Cannot convert '' to a command type which item 'StudyRoom_Light_Switch1' accepts: [OnOffType, RefreshType].
```

I tested this by creating a new thing (mqtt item) and a new item. Being newly created, it hasn't received the state update from mqtt and thus started off as NULL. I then ran the rule as above.

**Comment by boc-tothefuture** [comment]
So for this, we would be saying the opposite of UNDEF or NULL is ON?

**Comment by jimtng** [comment]
Alternatively maybe add a `toggle()` method?

**Comment by boc-tothefuture** [comment]
Resolved by 2.2.1
